frontend/src/components/GroupsAccordion.tsx

```diff
@@ -27,17 +28,29 @@ export const GroupsAccordionItem = ({
     <AccordionItem value={registrationName}>
       <AccordionTrigger className="px-4 hover:no-underline">
         <div className="flex items-center gap-4">
-          <Button
+          <div
```

Comment:
Nie lepiej zostawić button, a nie sztucznie robić z diva button?

Reply:
teoretycznie to co zrobiłeś nie naprawia tego błędu, tylko go ukrywa, nadal to jest bardzo średnio poprawny html i a11y przy tym leży, ten przycisk do usuwania powinien być na lewo od triggera po prostu

Reply:
true, przyklepalem gowno zeby nie smierdzialo